kong/api/routes/certificates.lua

```diff
@@ -12,6 +12,37 @@ return {
         self.params.snis = nil
       end
 
+      -- dont add the certificate or any snis if we have an SNI conflict
+      -- its fairly inefficient that we have to loop twice over the datastore
+      -- but no support for OR queries means we gotsta!
+      local seen = {}
+      if type(snis) == "table" then
```

Comment:
It seems this check is extraneous here. Following the beginning of this function, I believe `snis` can either be `nil` or a table. Hence, `if snis then` seems simpler and reasonable enough.